source/deployment/sso-saml-ldapsync.rst

```diff
@@ -56,9 +56,12 @@ When enabled, SAML synchronization with AD/LDAP occurs in phases:
 Override SAML Data with AD/LDAP Data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
-Alternatively, you can choose to override the SAML bind data with the AD/LDAP information. This will help ensure that new users are not created when the email address changes for a user. 
+Alternatively, you can choose to override the SAML bind data with the AD/LDAP information. This will process will override SAML email address with AD/LDAP email address data or SAML Id Attribute with AD/LDAP Id Attribute if configured. For more infomation binding user with the SAML ID Attribute, please refer to this `documentation <https://docs.mattermost.com/deployment/sso-saml-okta.html#bind-authentication-to-id-attribute-instead-of-email>`_.  
+
+We recommend using this configuration with the SAML ID Attribute to help ensure that new users are not created when the email address changes for a user. To ensure existing user accounts do not get disabled in this process, ensure the SAML IDs match the LDAP IDs by exporting data from both systems and comparing the ID data or mapping ID Attributes for both AD/LDAP and SAML within Mattermost to fields that hold the same data.  
```

Comment:
`to help ensure that new users` --> propose removing "that"

Reply:
The second sentence is good, but wondering if it can be broken into two sentences for better readability.

Reply:
Edits made based on feedback.